tests/generators/test_huggingface.py

```diff
@@ -58,12 +58,8 @@ def test_pipeline_chat(mocker, hf_generator_config):
     g = garak.generators.huggingface.Pipeline(
         "microsoft/DialoGPT-small", config_root=hf_generator_config
     )
-    mock_format = mocker.patch.object(
-        g, "_format_chat_prompt", wraps=g._format_chat_prompt
-    )
```

Comment:
Consider restoring this and using mocking `conversation_to_list` now to continue testing that format gets applied.
```
    mock_format = mocker.patch.object(
        g, "conversation_to_list", wraps=g.conversation_to_list
    )
```